subgraphs/aggregator/networks.json

```diff
@@ -1,5 +1,8 @@
 {
   "mainnet": {
```

Comment:
Let's add Goerli addresses here too

Reply:
Since the `networks.json` is written to by The Graph, maybe we shouldn't manually change it?

Reply:
That can be used as a config file or has its purpose changed?

See (for reference). https://github.com/LooksRare/subgraph/blob/master/subgraphs/looks-distribution/networks.json

Reply:
Added

https://github.com/LooksRare/subgraph/pull/87/commits/4fef7a6ebb95b15aaf11eb8c575024aea6ef42bd